FirebasePerformance.podspec

```diff
@@ -60,7 +60,6 @@ Firebase Performance library to measure performance of Mobile and Web Apps.
   s.dependency 'FirebaseInstallations', '~> 7.0'
   s.dependency 'FirebaseRemoteConfig', '~> 7.0'
   s.dependency 'GoogleDataTransport', '~> 8.2'
-  s.dependency 'GTMSessionFetcher/Core', '~> 1.1'
   s.dependency 'GoogleToolboxForMac/Logger', '~> 2.1'
```

Comment:
The `GoogleToolboxForMac` is the actual problem in #6597.  

GTMSessionFetcher is used by a few other Firebase libraries and doesn't cause similar problems.  Still great to remove to slim things down.

Reply:
Good catch! Cleaned up the dependency on GoogleToolboxForMac as well. Apparently, we are using the FIRLogging service and nothing specifically from GTM library.